chore(changelog): include indigo theme in the changelog (#7765)
IgniteUI/igniteui-angular: CHANGELOG.md

- `igxTreeGrid`
    - Removed `onDataPreLoad` event as it is specific for remote virtualization implementation, which is not supported for the `igxTreeGrid`. A more generic `onScroll` event is exposed and can be used instead.

### New Theme

> Ignite UI for Angular now have a new theme inspired by our own design system.

Reviewer: ```suggestion
Ignite UI for Angular now has a new theme based on our own design system.
```

> Depending on your use case you can use one of the following mixins:

Reviewer: ```suggestion
You can use one of the following mixins to include a dark or light indigo theme:
```

`igx-indigo-light-theme` and `igx-indigo-dark-theme`

We also added two new palettes that go with the new theme, `$light-indigo-palette` and `$dark-indigo-palette`.

> Next example shows how you can use the Indigo theme:

Reviewer: ```suggestion
The following example shows how you can use the Indigo theme:
```
